src/main/application/services.xml

```diff
@@ -36,9 +36,11 @@
                 <models>
                     <model name="colbert_encoder">
                         <intraop-threads>4</intraop-threads>
```

Comment:
Should we remove this? I don' think that intraop threads is something that is used when running with GPU provider?